src/test/java/com/minecolonies/entity/ai/util/ChatSpamFilterTest.java

```diff
@@ -10,89 +13,98 @@
 
 public class ChatSpamFilterTest
 {
+    @Mock
+    private EntityCitizen citizen;
+
+    private ChatSpamFilter filter;
+
+    private static final String MESSAGE_1 = "Whatever";
+    private static final String MESSAGE_2 = "Whatever2";
+
+    @Before
+    public void setup() {
```

Comment:
does the before resetup on each new test? Or do we need something like @BeforeEach?